linetools/spectra/lsf.py

```diff
@@ -213,6 +230,143 @@ def load_COS_data(self):
         
         return pixel_scale, data
 
+    def load_STIS_data(self):
+        """Load the right data according to `instr_config` for HST/STIS
+        instrument"""
+
+        # define pixel scales; values obtained from STScI
+        # these values must be consistent with the given LSFs
+        pixel_scale_dict = {
+            'G140L': 0.60 * u.AA,
+            'G140M': 0.05 * u.AA,
+            'G230L': 1.58 * u.AA,
+            'G230M': 0.09 * u.AA,
+            'E140M': 'lambda/91700',
+            'E140H': 'lambda/228000',
+            'E230M': 'lambda/60000',
+            'E230H': 'lambda/228000',
+            'G230LB': 1.35 * u.AA,
+            'G230MB': 0.15 * u.AA,
+            'G430L': 2.73 * u.AA,
+            'G430M': 0.28 * u.AA,
+            'G750L': 4.92 * u.AA,
+            'G750M': 0.56 * u.AA
+        }
+        # define channel based on grating name
+        channel_dict = {
+            'G140L': 'FUV-MAMA',
+            'G140M': 'FUV-MAMA',
+            'G230L': 'NUV-MAMA',
+            'G230M': 'NUV_MAMA',
+            'E140M': 'FUV-MAMA',
+            'E140H': 'FUV-MAMA',
+            'E230M': 'NUV-MAMA',
+            'E230H': 'NUV-MAMA',
+            'G230LB': 'CCD',
+            'G230MB': 'CCD',
+            'G430L': 'CCD',
+            'G430M': 'CCD',
+            'G750L': 'CCD',
+            'G750M': 'CCD'
+        }
+        # also need slits
+        available_slits = {
+            'G140L': ['52x0.1', '52x0.2', '52x0.5', '52x2.0'],
```

Comment:
Thanks!  I now get a different error when initializing the LSF object that I will append at the end of this message.  However, this does give me an object back, and the attached plot shows a comparison between the linetools LSF and the tabulated one.  Perhaps the odd wings are related to the error?

![stis_lsf_compare](https://cloud.githubusercontent.com/assets/13800698/18587630/fc7f9a62-7bf0-11e6-9063-4dae719bc63e.png)

`In [38]: lf1 = LSF(dict(name='STIS', grating='G140M',slit='52x0.1'))
---------------------------------------------------------------------------
IndexError                                Traceback (most recent call last)
<ipython-input-38-72df34ef4f3b> in <module>()
----> 1 lf1 = LSF(dict(name='STIS', grating='G140M',slit='52x0.1'))

/Users/jnburchett/Dropbox/Code/linetools/linetools/spectra/lsf.py in __init__(self, instr_config)
     53             self.pixel_scale, self._data = self.load_COS_data()
     54         elif self.name == 'STIS':
---> 55             self.pixel_scale, self._data = self.load_STIS_data()
     56         # IMPORTANT: make sure that LSFs are given in linear wavelength scales !!!
     57 

/Users/jnburchett/Dropbox/Code/linetools/linetools/spectra/lsf.py in load_STIS_data(self)
    348         # This new rel_pix grid will apply to all the wavelengths
    349 
--> 350         rel_pix = kernels_dict[wa_names[0]]['rel_pix']  # use the first one as reference
    351         rel_pix = np.linspace(-1*np.max(rel_pix), np.max(rel_pix), len(rel_pix))  # impose them linear and symmetric
    352         rel_pix = self.check_and_reformat_relpix(rel_pix)  # again just in case something odd happened

IndexError: index 0 is out of bounds for axis 0 with size 0`